build.gradle

```diff
@@ -117,7 +117,18 @@ subprojects {
 
     apply plugin: 'com.google.protobuf'
     apply plugin: 'java'
+
     apply plugin: 'org.spine3.tools.javadoc-checker'
+    // Javadoc checker plugin allows finding FQN links with the wrong format.
+    //"threshold" - is a number of wrong links that should be found to trigger a response.
+    //"responseType" - can be either "error" or "warn"
+    //      "warn" type allows to see all errors in build result records without failing it.
+    //      "error" type will fail the build process.
+    //      For more details please see org.spine3.tools.javadoc-checker.Response
+    checkJavadoc {
+        threshold = 0
```

Comment:
According to your description, we need `0` Javadoc style violations to trigger the response.
As I understand this, this is a number of "bearable" violations — the ones we __can__ live with. If it is so, please revisit the description comment.